packages/fluentui/react-bindings/test/styles/resolveStyles-test.ts

```diff
@@ -260,4 +261,92 @@ describe('resolveStyles', () => {
+
+    const ltrStyles = resolveStyles(ltrOptions, resolvedVariables, renderStyles);
+    const rtlStyles = resolveStyles(rtlOptions, resolvedVariables, renderStyles);
+
+    expect(ltrStyles).toHaveProperty('resolvedStyles.root.content', expect.stringMatching(/rtl:false/));
+    expect(ltrStyles).toHaveProperty('classes.root', expect.stringMatching(/rtl:false/));
+    expect(renderStyles).toHaveBeenCalledTimes(1);
+
+    expect(rtlStyles).toHaveProperty('resolvedStyles.root.content', expect.stringMatching(/rtl:true/));
+    expect(rtlStyles).toHaveProperty('classes.root', expect.stringMatching(/rtl:true/));
+    expect(renderStyles).toHaveBeenCalledTimes(2);
+  });
+
+  describe('enableBooleanVariablesCaching', () => {
+    test('avoids "classes" computation when enabled', () => {
+      const renderStyles = jest.fn().mockReturnValue('a');
+      const options = resolveStylesOptions({
+        props: { variables: { isFoo: true, isBar: null, isBaz: undefined } },
+        performance: { enableBooleanVariablesCaching: true }
+      });
+
+      expect(resolveStyles(options, resolvedVariables, renderStyles)).toHaveProperty('classes.root', 'a');
+      expect(resolveStyles(options, resolvedVariables, renderStyles)).toHaveProperty('classes.root', 'a');
+      expect(renderStyles).toHaveBeenCalledTimes(1);
+    });
+
+    test('forces "classes" computation when disabled', () => {
+      const renderStyles = jest.fn().mockReturnValue('a');
+      const options = resolveStylesOptions({
+        props: { variables: { isFoo: true, isBar: null, isBaz: undefined } },
+        performance: { enableBooleanVariablesCaching: false }
+      });
+
+      expect(resolveStyles(options, resolvedVariables, renderStyles)).toHaveProperty('classes.root', 'a');
+      expect(resolveStyles(options, resolvedVariables, renderStyles)).toHaveProperty('classes.root', 'a');
+      expect(renderStyles).toHaveBeenCalledTimes(2);
+    });
+
+    test('avoids "styles" computation when enabled', () => {
+      spyOn(componentStyles, 'root').and.callThrough();
+      const options = resolveStylesOptions({
+        props: { variables: { isFoo: true, isBar: null, isBaz: undefined } },
+        performance: { enableBooleanVariablesCaching: true }
+      });
+
+      expect(resolveStyles(options, resolvedVariables)).toHaveProperty('resolvedStyles.root');
+      expect(resolveStyles(options, resolvedVariables)).toHaveProperty('resolvedStyles.root');
+      expect(componentStyles.root).toHaveBeenCalledTimes(1);
+    });
+
+    test('requires "enableStylesCaching" to be enabled', () => {
+      const options = resolveStylesOptions({
+        performance: { enableStylesCaching: false, enableBooleanVariablesCaching: true }
+      });
+
+      expect(() => resolveStyles(options, resolvedVariables)).toThrowError(/Please check your "performance" settings on "Provider"/);
+    });
+
+    test('when enabled only plain objects can be passed as "variables"', () => {
```

Comment:
Are these last two tests still valid?

Reply:
Yep, I changed their contents and updated titles